extensions/test-src/org/pentaho/platform/plugin/services/exporter/PentahoPlatformExporterTest.java

```diff
@@ -42,9 +57,16 @@ public void setUp() throws Exception {
     scheduler = mock( IScheduler.class );
     session = mock( IPentahoSession.class );
     PentahoSessionHolder.setSession( session );
-    exporter = spy( new PentahoPlatformExporter( repo ) );
-    exporter.setScheduler( scheduler );
+    exporterSpy = spy( new PentahoPlatformExporter( repo ) );
+    exporterSpy.setScheduler( scheduler );
     doReturn( "session name" ).when( session ).getName();
+    IUnifiedRepository repository = mock( IUnifiedRepository.class );
```

Comment:
why have duplicate objects like exporter & exporterSpy and repo and repository? looks like they are configured the same way.